index.js

```diff
@@ -20,13 +20,25 @@ const missingBinaryError = async (command, arguments_) => {
 	}
 };
 
-const windowsKill = (input, options) => {
+const killWithTask = (input, options) => {
 	return taskkill(input, {
 		force: options.force,
 		tree: typeof options.tree === 'undefined' ? true : options.tree
 	});
 };
 
+const windowsKill = async (input, options) => {
+	try {
+		const firstTry = input;
+		return await killWithTask(firstTry, options);
+	} catch (error) {
+		if (typeof input === 'string') {
```

Comment:
You are swallowing the error if it's not a string. When fixing this, also add a regression test.